.github/workflows/continuous-integration.yml

```diff
@@ -34,7 +34,7 @@ jobs:
           restore-keys: |
             npm-
       - env:
-          NODE_AUTH_TOKEN: ${{ secrets.DOTTBOTT_TOKEN }}
+          NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
ok, so this repo needs to use a `GITHUB_TOKEN` as defined here https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-npm-registry in order to install packages in the ghithub npm registry (even public ones).